testcontainers/src/core/containers/request.rs

```diff
@@ -95,6 +98,10 @@ impl<I: Image> ContainerRequest<I> {
         self.ports.as_ref()
     }
 
+    pub fn ulimits(&self) -> Option<&Vec<ResourcesUlimits>> {
```

Comment:
We don't re-expose bollard's types right now (probably will in the future, with the access to client)

For now I'd prefer not to expose it with public methods. Any change of bollard would mean breaking change for us, when we could re-map changes into our structures in compatible way (depending on the changes)

Also, we don't re-export the type or crate, it may be not that convenient for users.

So for more we have two options:
- use our own type
- don't expose publicly, only internal use

Reply:
Makes sense! I created our own type. Let me know what you think.

The method now takes an `Option<i64>` for the `hard` limit since apparently that is optional, it'll use `soft` when it's missing. See https://docs.docker.com/reference/cli/docker/container/run/#ulimit